dts/bindings/gpio/ti,tca9538-gpio.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2018, Aapo Vienamo
+# Copyright (c) 2021, Laird Connectivity
+# SPDX-License-Identifier: Apache-2.0
+
+description: TCA9538 GPIO node
+
+compatible: "ti,tca9538"
+
+include: [i2c-device.yaml, gpio-controller.yaml]
+
+properties:
+    label:
+      required: true
```

Comment:
Why?

Reply:
Seems to be consistent with other YAML files where this appears. Can be set to false if needed.

Reply:
Most bindings are doing that because they are legacy from when we did not have DEVICE_DT_GET. It's fine to require this if you want to force all users to define labels, but now that device_get_binding is arguably the worst possible way to get a device, I think it makes sense not to require labels anymore. It will save some boilerplate in overlay files.

Again, not blocking though.